src/rs/scramble/scramble_search.rs

```diff
@@ -76,19 +79,16 @@ pub(crate) fn filtered_search(
     {
         return None;
     }
-    Some(
-        idfs.search(
-            scramble_pattern,
-            IndividualSearchOptions {
-                min_num_solutions: Some(1),
-                min_depth: min_scramble_moves,
-                max_depth: None,
-                disallowed_initial_quanta: None,
-                disallowed_final_quanta: None,
-            },
-        )
-        .next()
-        .unwrap()
-        .invert(),
+    idfs.search(
+        scramble_pattern,
+        IndividualSearchOptions {
+            min_num_solutions: Some(1),
+            min_depth: min_scramble_moves,
+            max_depth: max_scramble_moves.map(|n| n + 1),
```

Comment:
Actually, this is no longer needed (due to the functionality `OrbitOrientationConstraint::SetPieceZeroToIgnoredOrientation`). I'd rather remove it for now, as I don't know if all states have an 11-move solution when the search is also set to a minimum of 11 moves.